<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LP-5CD7485ZW3" ignored="0" name="Failure Test" tests="1" failures="1" timestamp="2020-08-11T00:27:58 IST" time="0.005" errors="0">
  <testcase name="doLogin" time="0.001" classname="listenerTests.TestFailure">
    <failure type="java.lang.AssertionError" message="Failing the Login Test">
      <![CDATA[java.lang.AssertionError: Failing the Login Test
at listenerTests.TestFailure.doLogin(TestFailure.java:14)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doLogin -->
</testsuite> <!-- Failure Test -->
